task/utils.go

```diff
@@ -0,0 +1,32 @@
+package task
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"k8s.io/client-go/rest"
+)
+
+const (
+	QPSRate   = "QPS_RATE"
+	BurstRate = "BURST_RATE"
```

Comment:
Can we prefix the env vars so that it is specific to this library?
For instance, `KUBERNETES_OPS_QPS_RATE`?